src/components/EllipsisValue/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import style from './style.css'
+import classnames from 'classnames'
+
+class FieldValue extends React.Component {
```

Comment:
I think component should be renamed to `EllipsisValue`

Reply:
Done.